content/en/docs/devops-user-guide/examples/a-maven-project.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "How to build and deploy a maven project"
+keywords: 'kubernetes, docker, devops, jenkins, maven'
+description: ''
+linkTitle: "Build And Deploy A Maven Project"
+weight: 200
+---
+
+## Prerequisites
+
+- You need to [enable KubeSphere DevOps System](../../../../docs/pluggable-components/devops/).
+- You need to create [DockerHub](http://www.dockerhub.com/) account.
+- You need to create a workspace, a DevOps project, and a user account, and this account needs to be invited into the DevOps project as the role of maintainer.
+- You need to be familiar with basic operations, and this document will not give a 101 introduction.
```

Comment:
Please be more specific here. Basic operation of what? Pipelines, Jenkins or Maven?

Reply:
remove it